tikv-configuration-file.md

```diff
@@ -1108,7 +1108,7 @@ Configuration items related to `rocksdb.defaultcf.titan`
 
 ## rocksdb.writecf
 
-Configuration items related to `rocksdb.writecf`
+Configuration items related to `rocksdb.writecf`, which are identical to `rocksdb.defaultcf`. Those not listed here have the same default value as `rocksdb.defaultcf`.
```

Comment:
What does it mean by "Those not listed here"? Are there any config items that are not documented here?

Reply:
Yes, they are the same as rocksdb.defaultcf. Listing all of the duplicates would be cumbersome.

Reply:
I don't think so. Suppose that you want to see a TiKV config item under `rocksdb.writecf` but are told to see the config items under `rocksdb.defaultcf` (another section) because they are duplicated and, therefore, documented only in one place. Both document completeness and user experience are compromised. One possible solution to that is to document all the config items in one place but the title should incorporate both `rocksdb.writecf and `rocksdb.defaultcf`. A note should also be given to explain why they are put together.

Reply:
Make sense.